models.py

```diff
@@ -83,32 +97,51 @@ def forward(self, x):
 
 class ByteCNNEncoder(nn.Module):
     def __init__(self, n, emsize, vocab_size, normalization, padding_idx=None,
-                 use_linear_layers=True, compress_channels=None):
+                 use_linear_layers=True, compress_channels=None,
+                 dropout=0.0, use_external_batch_norm=False, max_r=None):
```

Comment:
zamiast `max_r` nazwijmy to `external_batch_r`, bo się nie połapiemy za miesiąc, że `max_r` ma coś wspólnego z `use_external_batch_norm`.